qa/L0_onnx_execution_provider/test.sh

```diff
@@ -0,0 +1,151 @@
+    (cd models/onnx_float32_float32_float32_openvino && \
+            sed -i 's/^name: "onnx_float32_float32_float32_def"/name: "onnx_float32_float32_float32_openvino"/' \
+                config.pbtxt && \
+            echo "optimization { execution_accelerators { cpu_execution_accelerator : [\"openvino\"] } }" >> config.pbtxt) && \
+    # CPU execution accelerators on CPU context
+    cp -r models/onnx_float32_float32_float32_openvino models/onnx_float32_float32_float32_cpu_openvino && \
+    (cd models/onnx_float32_float32_float32_cpu_openvino && \
+            sed -i 's/^name: "onnx_float32_float32_float32_openvino"/name: "onnx_float32_float32_float32_cpu_openvino"/' \
+                config.pbtxt && \
+            echo "instance_group [ { kind: KIND_CPU }]" >> config.pbtxt) && \
+    # Unknown GPU execution accelerator
+    cp -r models/onnx_float32_float32_float32_def models/onnx_float32_float32_float32_unknown_gpu && \
+    (cd models/onnx_float32_float32_float32_unknown_gpu && \
+            sed -i 's/^name: "onnx_float32_float32_float32_def"/name: "onnx_float32_float32_float32_unknown_gpu"/' \
+                config.pbtxt && \
+            echo "optimization { execution_accelerators { gpu_execution_accelerator : [\"unknown_gpu\"] } }" >> config.pbtxt) && \
+    # Unknown CPU execution accelerators
+    cp -r models/onnx_float32_float32_float32_def models/onnx_float32_float32_float32_unknown_cpu && \
+    (cd models/onnx_float32_float32_float32_unknown_cpu && \
+            sed -i 's/^name: "onnx_float32_float32_float32_def"/name: "onnx_float32_float32_float32_unknown_cpu"/' \
+                config.pbtxt && \
+            echo "optimization { execution_accelerators { cpu_execution_accelerator : [\"unknown_cpu\"] } }" >> config.pbtxt)
+
+run_server
+if [ "$SERVER_PID" == "0" ]; then
+    echo -e "\n***\n*** Failed to start $SERVER\n***"
+    cat $SERVER_LOG
+    exit 1
+fi
+
+RET=0
+
+set +e
+
+grep "TensorRT Execution Accelerator is set for onnx_float32_float32_float32_trt" $SERVER_LOG
+if [ $? -ne 0 ]; then
+    echo -e "\n***\n*** Failed. Expected TensorRT Execution Accelerator is set\n***"
+    RET=1
+fi
+grep "CUDA Execution Accelerator is set for onnx_float32_float32_float32_trt" $SERVER_LOG
+if [ $? -ne 0 ]; then
+    echo -e "\n***\n*** Failed. Expected CUDA Execution Accelerator is set\n***"
+    RET=1
+fi
+
+grep "OpenVINO Execution Accelerator is not supported for onnx_float32_float32_float32_openvino" $SERVER_LOG
+if [ $? -ne 0 ]; then
+    echo -e "\n***\n*** Failed. Expected OpenVINO is not supported\n***"
+    RET=1
+fi
+grep "CUDA Execution Accelerator is set for onnx_float32_float32_float32_openvino" $SERVER_LOG
+if [ $? -ne 0 ]; then
+    echo -e "\n***\n*** Failed. Expected CUDA Execution Accelerator is set\n***"
+    RET=1
+fi
+
+grep "OpenVINO Execution Accelerator is not supported for onnx_float32_float32_float32_cpu_openvino" $SERVER_LOG
```

Comment:
Why isn't openvino supported for a CPU context?

Reply:
This PR hasn't included build with OpenVINO, will work on OpenVINO and change this test accordingly in another PR.